Sources/MobileCoinClient+Deprecated.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+// swiftlint:disable function_default_parameter_at_end multiline_function_chains let_var_whitespace
```

Comment:
we going  to fix these in a subsequent PR or will they be allowed to stay?

Reply:
no, Im making some exceptions for specific files based on their content.